package cn.datai.gift.utils.exception;

/**
 * 用户正在支付
 * Created by zhangyutao on 2017/5/17.
 */
public class PayingException extends RuntimeException {
}
